### PR TITLE
Speedup of theme mode loading

### PR DESCRIPTION
**Pull request by lomion0815**
By default the dark theme was rendered. After the complete page was loaded the theme was switched according the stored value. This cause a flicker (background was first black for a second (until the page was completely loaded) and then turned white.

This commit moves the loading of the theme mode to the beginning (into the <head> section). Therefore the correct mode gets rendered.

### CODE REVIEW

File: layouts/partials/head.html [hunk 1]
```diff
@@ -11,11 +11,21 @@
...
+<!-- Load theme mode from browser cache -->
+<script>
+	var darkTheme = document.getElementById("dark-mode-theme");
+	var storedTheme = localStorage.getItem("dark-mode-storage");
+	if ( storedTheme === "dark") {
+		darkTheme.disabled = false;
+    } else if (storedTheme === "light") {
+		darkTheme.disabled = true;
+    }
+</script>
```
Comment (matsuyoshi30) [inline layouts/partials/head.html:28]: Please remove unnecessary whitespace in `if` clause and fix indent. (I know I should prepare linter...)

**Review by matsuyoshi30 — CHANGES_REQUESTED** [review]

File: layouts/partials/head.html (hunk 1 above)
Comment (lomion0815) [reply layouts/partials/head.html]: Done!

**Comment by matsuyoshi30** [comment]
@lomion0815 Great PR! Thank you.
